puppet/stdlib/tests/file_line.pp

```diff
@@ -2,8 +2,8 @@
 # of the file_line resource type.
 file { '/tmp/dansfile':
   ensure => present
-}->
-file_line { 'dans_line':
+}
+-> file_line { 'dans_line':
```

Comment:
The files under `puppet/stdlib` and `puppet/apt` are third-party code that we should just exclude from this linter.